modules/installation-osp-creating-control-plane-ignition.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+[id="installation-osp-creating-control-plane-ignition_{context}"]
+= Creating control plane ignition files
+
+Installing {product-title} on {rh-openstack-first} on your own infrastructure requires control plane Ignition files. You must create multiple ignition files. 
+
+[NOTE]
+As with the bootstrap Ignition configuration, you must ensure that the hostname per control plane machine is set to an expected value.
+
+.Prerequisites
+
+* The infrastructure ID from the installation program's metadata file is set as an environment variable (`$INFRA_ID`)
```

Comment:
see common.yaml

Reply:
Actually, my inclination is to keep this in. The user wouldn't necessarily have common.yaml at this point.

Reply:
Again, the Python script below uses the environment variable :+1: